mysite/settings.py

```diff
@@ -52,6 +52,7 @@
         'DIRS': [],
         'APP_DIRS': True,
         'OPTIONS': {
+            'debug': True,  # Only enable this during testing
```

Comment:
It is customary to mark these as
```suggestion
            'debug': True,  # FIXME: Only enable this during testing
```
Since many tools are then able to collect this information. Although, you may have to ignore this line for pylint.

Reply:
Okay